athenahealth/appointments.go

```diff
@@ -622,6 +626,54 @@ func (h *HTTPClient) BookAppointment(ctx context.Context, patientID, apptID stri
 	return out[0], nil
 }
 
+type UpdateBookedAppointmentOptions struct {
+	// New appointment type ID for this appointment.
+	AppointmentTypeID *string `json:"appointmenttypeid"`
+	// New department ID for this appointment.
+	DepartmentID *string `json:"departmentid"`
+	// New provider ID for this appointment.
+	ProviderID *string `json:"providerid"`
+	// New supervisingprovider ID for this appointment.
+	SupervisingProviderID *string `json:"supervisingproviderid"`
+}
+
+// UpdateBookedAppointment
+// PUT /v1/{practiceid}/appointments/booked/{appointmentid}
+// https://docs.athenahealth.com/api/api-ref/appointment-booked#Appointment-Booked
+// Output Parameters
+// status	string	This subroutine will return 1 on success, and will otherwise return an error message.
```

Comment:
We should remove this comment or keep it with the declaration of the success response variable (see comment below) - since this method only returns an error or `nil` it would be misleading as documentation here